website_docs/manual.md

```diff
@@ -7,9 +7,88 @@ aliases: [/docs/instrumentation/go/instrumentation, /docs/instrumentation/go/man
 
 Instrumentation is the process of adding observability code to your application. There are two general types of instrumentation - automatic, and manual - and you should be familiar with both in order to effectively instrument your software.
 
+## Initializing a Tracer
+
+To create spans, you'll need to initialize a tracer first. Some important things to consider:
+
+* Tracers should live at the module level and have a lifecycle congruent to an app's lifecycle
+* There should only be one tracer per module
```

Comment:
```suggestion
* There should only be one tracer per package
```

One `Tracer` per module is almost certainly not correct.  One `Tracer` per package is possibly correct, but I'm not sure I would even go as far as to say there should only be one per package.  That may be the most common scenario, but there are absolutely scenarios where multiple `Tracer`s, and even `TracerProvider`s are needed from a single package and I'm not sure we should be prescriptive about their cardinality.